quickstart.rst

```diff
@@ -0,0 +1,57 @@
+.. _quickstart:
+
+Quick Start
+===========
+
+The easiest way to get started with Multinet is to go to https://multinet.app,
+and begin experimenting.
+
+To build a local development environment, you will need to *install
+dependencies*, *launch a database*, *build and run the server*, and *build and
+run the client*.  Follow these steps:
+
+Install Dependencies
+--------------------
+
+These projects are built using the following tools; ensure that you have them
+installed:
+- Docker
+- Python 3.7
+- Pipenv
+- Node
+- Yarn
+
+Build and Run ``multinet-server``
+---------------------------------
+
+1. Clone the repository: ``git clone https://github.com/multinet-app/multinet-server``.
+2. Move into the repository: ``cd multinet-server``.
+3. Copy the ``.env.default`` file: ``cp .env.default .env`` (or symlink it: ``ln
+   -s .env.default .env``).
+4. Install the Python dependencies: ``pipenv install``.
+5. Activate the virtual environment: ``pipenv shell``.
```

Comment:
Is this necessary?

Reply:
Activating the shell? Yes, at least for the quick start instructions. it pulls in the .env file and sets up the serve port, which can be done manually, but this is the fastest way to get up and running.

Reply:
But running the next step by itself does that, doesn't it? The `pipenv run serve` step.

Reply:
You're right! I thought for some reason this wasn't the case but good catch. See 
dbb9c73.